### ISSUE TITLE
Presence of both `access_id` and `access_url` in a response

### ISSUE THREAD

**Issue by hannes-ucsc** [ga4gh/data-repository-service-schemas#360]
I interpret this clause of the spec

https://github.com/ga4gh/data-repository-service-schemas/blob/2425439e509b252a76d2a85b86d9fa26747addfc/openapi/data_repository_service.swagger.yaml#L343

to mean that having both `access_id` and `access_url` in a response is acceptable. Is that the case? 

If so, what is the point of providing an access_id and following it in a separate request, when a URL is already available?

**Comment by trbica** [comment]
Providing both `access_id` and `access_url` in a response is misleading for DRS API client and is not aligned with [Fetching DRS Objects](https://ga4gh.github.io/data-repository-service-schemas/preview/develop/docs/#section/Fetching-DRS-Objects) section. DRS API client MUST only work with `access_url` to access DRS object content.
I would also like to hear more about resolving this as it is creating issues on the consumer side.

**Comment by nmalfroy** [comment]
I would argue the documentation that @hannes-ucsc points explicitly states that both are legitimate.  Also, for signed URLs, then you would definitely not use `access_url`

**Comment by hannes-ucsc** [comment]
I'd consider the possibility that the phrase "at least one of" was a mistake and that the intent was "either one of".

**Comment by nmalfroy** [comment]
Yeah, I agree...it appears that this may be a bug in the spec wording...which would then break our implementation
